data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,279 @@ event "remnant: pantica void sprites"
+			choice
+				`	"Have you asked someone with more experience for advice?"`
+				`	"Don't worry about it."`
+					goto worry
+				`	"Can I help with something?"`
+					goto help
+			`	"I did. The senior researcher told me that most of the team feels something is not quite right here. They say there are some potential theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
+				goto help
+			label worry
+			`	"Thanks for the advice, but that is our purpose here - to worry about all the things that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
+			label help
+			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A* to collect some fresh data I can use to check our records, and I could use a transport..."`
+			choice
+				`	"I don't know where that is."`
+					goto center
+				`	"Isn't that the center of the galaxy?"`
+					goto centeralt
+				`	"I have been there."`
+			`	"Really?" She looks surprised. "I have seen sensor logs and recordings from previous visits, but not many people have actually been there." She pauses to take a bite from her neglected meal. "Being able to take part in these fieldwork expeditions is something I have been looking forward to for ages, so of course Sagittarius A* is the first place on my list."`
+				goto center
+			label centeralt
+			`	"Yes, that is correct." She signs firmly.`
+			label center
+			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
+			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
+			choice
+				`	"Sure, what do you need?"`
+					goto scienceequipment
+				`	"You want me to take you to a black hole?!"`
+				`	"Sorry, I'm not interested in visiting a black hole."`
+					decline
+			`	She looks momentarily surprised by your reaction. "Yes, exactly that. Current gravitational repulsor engines can nullify most of the gravitational pull, and an extra thermoelectric cooler or two should be sufficient to manage the heat. As such, the risks are mangeable so long as you do not try to approach the event horizon.`
+			label scienceequipment
+			`	"For this mission in specific, however, we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require six outfit space, and requires a scientist, so you will need a free bunk for them, too.`
```

Comment:
```suggestion
			`	"For this mission in specific, however, we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require six tons of outfit space and a scientist to run it, so you will need a free bunk for them, too.`
```